data/nmstate/scc.yaml

```diff
@@ -0,0 +1,20 @@
+{{ if .EnableSCC }}
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: nmstate
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+allowHostNetwork: true
+allowHostIPC: false
+allowHostPID: false
+allowHostPorts: false
+readOnlyRootFilesystem: false
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+users:
+- system:serviceaccount:nmstate:nmstate-handler
```

Comment:
The `nmstate` part is the namespace. We need to parametrize it

Reply:
Interesting this is wrong at kubernetes-nmstate repo :-(

Reply:
we need this https://github.com/nmstate/kubernetes-nmstate/pull/581